worker-queue-rabbit/src/main/java/com/hpe/caf/worker/queue/rabbit/RabbitWorkerQueue.java

```diff
@@ -337,12 +325,12 @@ private void createConnection(TaskCallback callback, WorkerConfirmListener liste
         ((Recoverable)conn).addRecoveryListener(new WorkerConnectionListener(callback, listener));
     }
 
-    private void declareWorkerQueue(Channel channel, String queueName, int maxPriority)
+    private void declareWorkerQueue(Channel channel, String queueName)
         throws IOException
     {
         if (!declaredQueues.contains(queueName)) {
 
-            RabbitUtil.declareWorkerQueue(channel, queueName, maxPriority);
+            RabbitUtil.declareWorkerQueue(channel, queueName, config.getMaxPriority());
```

Comment:
Since we decided the quorum queue change cannot be completed for a worker in isolation are we now not supporting both classic and quroum? If I remove this I am removing the ability to add priority when the queue is classic. Or maybe I am misunderstanding what you are saying here

Reply:
After talking it through we agreed to keep both declareWorkerQueue implementations and not remove max priority from the config.